autobuild.js

```diff
@@ -0,0 +1,39 @@
+const fs = require('fs');
+const readline = require('readline');
+
+const fromPath = './src/js/index.ts';
+const toChangePath = [
+  {
+    path: './package.json',
+    spaces: 2
+  },
+  {
+    path: './package-lock.json',
+    spaces: 2
+  },
+  {
+    path: './manifest.json',
+    spaces: 4
+  }
+]
+
+const readInterface = readline.createInterface({
+  input: fs.createReadStream(fromPath),
+  console: false
+});
+
+
+readInterface.on('line', function(line) {
```

Comment:
might it be easier / more reliable to `export` the version number and then `import` it?